### ISSUE TITLE
[Question] is it possible to have a webform send a copy to a designated email address?

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#3543]
Hi,

this may already be possible and I am just overlooking a settings option. When we create an online form, it sends the data to the (Leads in my case). As far as I can see, it doesn't send it to any particular email account. Ideally it would send a copy so that when we don't have access to yetiforce we still get a notification that a lead from a webpage has been sent. 

If this does not exist, is there a simple way to add this? I can't see the Web forms in edit fields section of the modules.

Regards

Percy

**Comment by florek41** [comment]
You can try do it with workflows (https://gitdeveloper.yetiforce.com/index.php?module=Workflows&parent=Settings&view=List&block=15&fieldid=24)
You need to set new workflow which send you email on every new created record in Leads module. Additionally you can add new field to Leads called "from webform" and filtered events in workflow only when this condition is meet.

If you need more help with that - let me know!

**Comment by PercyP** [comment]
@florek41 

your suggestion is exactly what I needed (I don't even need to create the new field, as want any lead to send a message to the assigned to person). Thank you! 

I have now created a workflow, which is to send a message when a new lead is added. I chose the:

 'Add task for workflow: Send Mail'

Then populated the message choosing the Assigned to as the recipient for the email. I added some content from the Leads for i.e. name, description, activated and saved the new workflow, then did a test. The lead goes into the database fine, but the Assigned to recipient doesn't receive the message. 

Is there a step I am missing here?
